how2meet/services/sms.py

```diff
@@ -0,0 +1,50 @@
+"""
+Twilio SMS service. Boilerplate and API calls to arbitrarily send SMS messages.
+"""
+
+import os
+
+from fastapi import FastAPI
+from twilio.rest import Client
+from twilio.rest.api.v2010.account.message import MessageInstance
+
+app = FastAPI()  # TODO: is this problematic? Need to be same object for all routes?
```

Comment:
I'm confused what the point of this file is I guess. Are you intending to define an API route to send the sms (and thus would need an instance of fastAPI), or are you simply defining the function to send the sms which will then later be called from another endpoint?

Reply:
I'd begun with the intention to define a route, but I don't actually feel strongly about that (so you can help decide if `services/*.py` files should be routes or simply utilities to be called elsewhere). What I really wanted to do was configure Twilio to see if I could get sms working at all. Clearly the code in this PR is super minimal--the vast majority of the work was the on the twilio console side of things.